lib/AST/CMakeLists.txt

```diff
@@ -55,11 +55,10 @@ add_swift_library(swiftAST STATIC
   TypeRepr.cpp
   TypeWalker.cpp
   USRGeneration.cpp
-  LINK_LIBRARIES
+  LINK_LIBS
     swiftMarkup
     swiftBasic
 
-  INTERFACE_LINK_LIBRARIES
```

Comment:
This is necessary to fix a Windows linker error - clangDriver links to version.lib. If we used LINK_LIBRARIES, as we did before, we wouldn't transitively get the version.lib symbols. 
with INTERFACE_LINK_LIBRARIES, we do

Do we still maintain this behaviour?